deny.toml

```diff
@@ -20,6 +20,8 @@ ignore = [
   "RUSTSEC-2020-0016",
   # TODO: Remove when rusoto-* no longer depends on dirs.
   "RUSTSEC-2020-0053",
+  # TODO: remove when fixed version available.
```

Comment:
nit: It might be useful to enforce issue numbers on todos here as well as part of a future PR.

Reply:
I'm leaving this one for now, because I'm currently wondering whether `cargo deny` is worth the effort &ndash; whenever it complains I just add exceptions (after a quick investigation to see whether there's an easy version upgrade to fix things).  WDYT?

Reply:
I am unsure about the value. I think it is useful to know about the issues, but just adding exceptions does not really solve anything. Perhaps it is fine to remove it from CI and run it on a schedule. The main problem with that is how to ensure that the result of the scheduled task is not just ignored, and to be able to detect new issues amongst the preexisting warnings and errors.